code/game/machinery/cryopod.dm

```diff
@@ -335,7 +339,7 @@
 
 		var/preserve = null
 		for(var/T in preserve_items)
-			if(istype(W,T))
+			if(istype(W,T) && !(W in do_not_preserve_items))
```

Comment:
Well, you could snowflake posibrains into the check, and cross-compare to occupant.internal_organs to see if it's theirs; Or check posibrain.owner, see if it's the occupant.